<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>1111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="italicButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/Italicimage.png</normaloff>:/Italicimage.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="lineButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/underLine.png</normaloff>:/underLine.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="boldButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/boldImage.png</normaloff>:/boldImage.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QToolButton" name="betweenAlignButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/betweenAlign.png</normaloff>:/betweenAlign.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="cancelLineButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/midLine.png</normaloff>:/midLine.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QToolButton" name="leftAlignButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/leftAlign.png</normaloff>:/leftAlign.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QToolButton" name="centerAlignButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/centerAlign.png</normaloff>:/centerAlign.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QToolButton" name="openFileButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/open.png</normaloff>:/open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QToolButton" name="colorButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/colorLine.png</normaloff>:/colorLine.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="fontLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>폰트</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="19">
       <widget class="QTextEdit" name="textEdit">
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QToolButton" name="saveButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/save.png</normaloff>:/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QToolButton" name="rightAlignButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/rightAlign.png</normaloff>:/rightAlign.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="sizeBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="separator"/>
    <addaction name="fileOpenAction"/>
    <addaction name="saveAction"/>
    <addaction name="otherNameSaveAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>보기</string>
    </property>
    <addaction name="plusAction"/>
    <addaction name="minusAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>도구</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="fileOpenAction">
   <property name="text">
    <string>열기</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
  <action name="otherNameSaveAction">
   <property name="text">
    <string>다른이름으로 저장</string>
   </property>
  </action>
  <action name="plusAction">
   <property name="text">
    <string>확대</string>
   </property>
  </action>
  <action name="minusAction">
   <property name="text">
    <string>축소</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
